app/assets/javascripts/code/controllers/_mobile_sessions_map_ctrl.js

```diff
@@ -91,14 +91,16 @@ export const MobileSessionsMapCtrl = (
   $scope.$watch("params.get('data').heat", function(newValue, oldValue) {
     console.log("watch - params.get('data').heat - ", newValue, " - ", oldValue);
     if (newValue === oldValue) return;
-    if (mobileSessions.noOfSelectedSessions() !== 0) {
```

Comment:
It would be awesome if we can clean up some day and remove all the logic that handles selecting multiple sessions. It adds unnecessary complexity and we don't need it anymore.